code/controllers/stonedmc/subsystem/dbcore.dm

```diff
@@ -138,15 +135,14 @@ SUBSYSTEM_DEF(dbcore)
 	query_round_start.Execute()
 	qdel(query_round_start)
 
+
 /datum/controller/subsystem/dbcore/proc/SetRoundEnd()
 	if(!Connect())
 		return
-	/*
-	var/sql_station_name = sanitizeSQL(station_name())
-	var/datum/DBQuery/query_round_end = SSdbcore.NewQuery("UPDATE [format_table_name("round")] SET end_datetime = Now(), game_mode_result = '[sanitizeSQL(SSticker.mode_result)]', station_name = '[sql_station_name]' WHERE id = [GLOB.round_id]")
+	var/datum/DBQuery/query_round_end = SSdbcore.NewQuery("UPDATE [format_table_name("round")] SET end_datetime = Now(), game_mode_result = '[sanitizeSQL(SSticker.mode.round_finished)]'")
```

Comment:
You've removed the where statement here.

Reply:
Good catch